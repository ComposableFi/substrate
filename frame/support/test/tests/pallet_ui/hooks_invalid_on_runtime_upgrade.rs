#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::*;
	use frame_system::pallet_prelude::*;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(_);

	#[pallet::hooks]
	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
		fn on_runtime_upgrade() -> Weight { 0 }
	}

	#[pallet::call]
	impl<T: Config> Pallet<T> {}
}

fn main() {
}
