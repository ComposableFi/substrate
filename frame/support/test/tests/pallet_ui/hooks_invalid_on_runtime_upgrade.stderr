error[E0407]: method `on_runtime_upgrade` is not a member of trait `Hooks`
  --> tests/pallet_ui/hooks_invalid_on_runtime_upgrade.rs:14:3
   |
14 |         fn on_runtime_upgrade() -> Weight { 0 }
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not a member of trait `Hooks`
